migrations/postgres/3_create_version2.down.sql

```diff
@@ -1,14 +1,24 @@
--- Copyright © 2024 Triad National Security, LLC. All rights reserved.
+-- MIT License
 --
--- This program was produced under U.S. Government contract 89233218CNA000001 for
--- Los Alamos National Laboratory (LANL), which is operated by Triad National
--- Security, LLC for the U.S. Department of Energy/National Nuclear Security
--- Administration. All rights in the program are reserved by Triad National
--- Security, LLC, and the U.S. Department of Energy/National Nuclear Security
--- Administration. The Government is granted for itself and others acting on its
--- behalf a nonexclusive, paid-up, irrevocable worldwide license in this material
--- to reproduce, prepare derivative works, distribute copies to the public,
--- perform publicly and display publicly, and to permit others to do so.
+-- Copyright © 2024 Contributors to the OpenCHAMI Project
```

Comment:
I think we can update the year to the 2025.

Reply:
I updated the copyrights on all of these files to 2024-2025.
The git history showed that the files were added in 2024.
Tell if you think the copyright dates should be something else.